lib/imapoptions

```diff
@@ -217,7 +217,7 @@ are listed with ``<none>''.
 /* Alternative INBOX spellings that can't be accessed in altnamespace
    otherwise go under here. */
 
-{ "annotation_db", "twoskip", STRINGLIST("skiplist", "twoskip", "zeroskip"), "3.1.6" }
+{ "annotation_db", "twoskip", STRINGLIST("skiplist", "twoskip"), "3.1.6" }
```

Comment:
Need to replace the version number with "UNRELEASED".  This applies to all the other foo_db option changes too, except for backup_db which I'll address separately

Reply:
done. please recheck.